Static/Section.swift

```diff
@@ -13,6 +13,9 @@ public struct Section: Hashable, Equatable {
         /// Custom view for the header or footer. The height will be the view's `bounds.height`.
         case view(UIView)
 
+        // View Sized with autolayout
+        case autoLayoutView(UIView)
```

Comment:
I'm definitely not tied to this naming since in my current case name, but wanted to convey this would handle automatic dimensioning rather than use the views set bounds.height. Only added a new case to prevent previous integration breaks.

My current use case adds stackViews without constraints, but ended up using this to not break existing .view(_) sizing.